en-US/focus-ios.xliff

```diff
@@ -760,6 +760,16 @@
         <target>SECURITY</target>
         <note>Header label for security toggles displayed in the settings menu</note>
       </trans-unit>
+      <trans-unit id="Settings.setAsDefaultBrowser">
+        <source>Set as Default Browser</source>
+        <target>Set as Default Browser</target>
+        <note>Label title for set as default browser row</note>
+      </trans-unit>
+      <trans-unit id="Settings.setAsDefaultBrowserDescription">
+        <source>Set links from websites, emails and messages to open automatically in Focus.</source>
```

Comment:
The app name should be passed as a variable, not hard-coded (and there should be a comment explaining what replaces the placeholder). See line 164 for example.

Focus is not Focus in DE/AT/CH.